### DISCUSSION TITLE
로깅 상세 스펙을 결정한다

### DISCUSSION THREAD

**Discussion by Hyeon9mak** [woowacourse/prolog#340] [Feature]
1. 프로필별(프로덕션, 테스트, ...) 로깅 레벨 범위
2. 프로필별(프로덕션, 테스트, ...) 로깅 파일 저장
    - 테스트는 로그 파일을 저장하지 않는다던지? 
3. 로깅 포맷(양식, 글자색 등)
4. 로깅 파일 분할 기준(날짜별, 용량별)
5. 로깅 파일이 쌓일 경우 처리 방식(S3로 이전 등)

**Comment by gracefulBrown** [comment]
환경별 레벨과 장소는 요정도는 어떤가요?
- 프로덕션: error, file, ~~console~~
- 개발: debug, file, ~~console~~
- 테스트: debug, console
- 로컬: debug, console

파일 분할은 날짜별 추천하고요
7일 지나면 날리는걸로 하면 어떨까요?

로깅 포맷은.. 여러분들의 미적 감각을 믿습니다 :)

**Reply by bperhaps** [reply]
프로덕션에서 콘솔 출력이 필요한가요? 콘솔 출력은 io stream이라서 syncronized가 걸려있기에 성능에 큰 영향이 있다고 알고 있어요 ㅜ

**Reply by qhals321** [reply]
오옹?? 파일로깅도 io stream 아니었어?! 궁금궁금!!
